### PR TITLE
aws 18.4.2

### PR DESCRIPTION
**Pull request by whites11**
### Checklist
- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### CODE REVIEW

**Comment by whites11** [comment]
tests are good (china is stupid), we can merge as soon as release notes are checked

File: aws/v18.4.2/announcement.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**Workload cluster release v18.4.2 for AWS is available**. This is a patch release that fixes a problem that caused CNI downtime when upgrading from v18 to v19. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v18.4.2/).
+
+> **_Info:_** Cilium support that was introduced in 18.0.0 was rolled back to AWS CNI because of an unresolved upstream bug. Cilium support is postponed to version v19.0.0.
```
Comment (njuettner) [inline aws/v18.4.2/announcement.md:3]: maybe we also add, there was an issue with the old sqs and event IAM permissions

***IAM Permissions Requirements***
The minimal requirement for the IAM permissions is [Version 3.3.0](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/blob/master/CHANGELOG.md#330---2023-05-11) of [giantswarm-aws-account-prerequisites](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/) repository.

**Review by njuettner — APPROVED** [review]

File: aws/v18.4.2/announcement.md (hunk 1 above)
Comment (T-Kukawka) [reply aws/v18.4.2/announcement.md:3]: yeah i will add this

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**